server/app/services/applicant/question/AddressQuestion.java

```diff
@@ -59,24 +70,13 @@ public ImmutableSet<ValidationErrorMessage> getQuestionErrors() {
     return errors.build();
   }
 
-  @Override
-  public ImmutableSet<ValidationErrorMessage> getAllTypeSpecificErrors() {
-    return ImmutableSet.<ValidationErrorMessage>builder()
-        .addAll(getAddressErrors())
-        .addAll(getStreetErrors())
-        .addAll(getCityErrors())
-        .addAll(getStateErrors())
-        .addAll(getZipErrors())
-        .build();
-  }
-
-  public ImmutableSet<ValidationErrorMessage> getAddressErrors() {
+  private ImmutableSet<ValidationErrorMessage> validateAddress() {
     // TODO: Implement address validation.
     return ImmutableSet.of();
   }
 
-  public ImmutableSet<ValidationErrorMessage> getStreetErrors() {
-    if (isAnswered() && getStreetValue().isEmpty()) {
```

Comment:
Here and in other QuestionImpl subclasses, I removed the call to `isAnswered` for each validator since we now call that in the base class prior to calling `getValidationErrorsInternal`.